docs/sidebars.js

```diff
@@ -190,33 +187,57 @@ const sidebars = {
     },
     {
       type: 'category',
-      label: 'Building services',
+      label: 'Architecture',
+      link: {
+        'type': 'generated-index',
+      },
+      items: [
+        {
+          type: 'doc',
+          label: 'Concepts',
+          id: 'architecture/concepts',
+        },
+        {
+          type: 'doc',
+          label: 'Coordinator',
+          id: 'architecture/coordinator',
+        },
+        {
+          type: 'doc',
+          label: 'Marbles',
+          id: 'architecture/marbles',
+        },
+        {
+          type: 'doc',
+          label: 'Keys and cryptography',
+          id: 'architecture/security',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Building Enclaves',
```

Comment:
```suggestion
      label: 'Building Marbles',
```
Not sure about the best word here.
* Service might be too generic, but easiest to understand for new users
* Marble is precise, but might confuse new users
* Building an Enclave is just a subtask of what this section is actually about, but might confuse new users less than Marble